reflex-algos/src/main/python/mlops/parallelm/mlops/constants.py

```diff
@@ -54,6 +54,7 @@ class MLOpsRestHandles:
     LOGIN = "auth/login"
     GROUPS = "groups"
```

Comment:
shouldnt we remove "GROUPS"?

@lior-amar are we going to make a one-time backward-incompatible move to 1.4?